src/rules/whitespace/sequential_blank.ts

```diff
@@ -39,7 +42,7 @@ export class SequentialBlank extends ABAPRule {
     let blanks = 0;
 
     for (let i = 0; i < rows.length; i++) {
-      if (rows[i] === "") {
+      if (SequentialBlank.isBlankOrWhitespace(rows[i])) {
```

Comment:
super minor thing, but this if-block could be shortened with:
```typescript
blanks = SequentialBlank.isBlankOrWhitespace(rows[i]) ? blanks++ : blanks = 0;
```

Reply:
I don't like having side effects in ternaries, the line does a bit too much. But I did improve the code a bit differently.